ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockReplicaId.java

```diff
@@ -97,6 +99,16 @@ public boolean isDown() {
   }
 
   @Override
+  public boolean isSealed() {
+    return isSealed;
+  }
+
+  @Override
+  public void setSealedState(boolean isSealed) {
+    this.isSealed = isSealed;
```

Comment:
wondering how this can be made to show up at the partition level (if all replicas get marked sealed). If we are providing a mock, it would be nice to keep it in sync with the originals to help with testing. Let's chat about this to see how hard/easy this is.

Reply:
Chatted about this, implemented partition "bubble-up" code